webapp/components/survey/NodeDefDetails/BasicProps/BasicProps.js

```diff
@@ -172,7 +172,11 @@ const BasicProps = (props) => {
 BasicProps.propTypes = {
   state: PropTypes.object.isRequired,
   Actions: PropTypes.object.isRequired,
-  editingFromDesigner: PropTypes.bool.isRequired,
+  editingFromDesigner: PropTypes.bool,
+}
+
+BasicProps.defaultProps = {
+  editingFromDesigner: null,
```

Comment:
should it be false by default?

Reply:
should it have a defaultProp at all ?

Reply:
i deleted because i put Boolean in useIsInRoute which return true or false